VideoAPI/VideoApi.UnitTests/Mappings/ConferenceToDetailsResponseMapperTests.cs

```diff
@@ -28,6 +28,7 @@ public void should_map_all_properties()
                 .Excluding(x => x.ConferenceStatuses)
                 .Excluding(x => x.State)
                 .Excluding(x => x.Tasks)
+                .Excluding(x => x.Messages)
```

Comment:
We do need to retrieve the messages when getting a conference. This should be included in mappings

Reply:
Hi Sunil, I haven't explicitly specified that commands and queries should be part of this story but it will be good to add them as part of this task. Could you add them please as they are essentially part of the domain work?